lux/test/unit/lux/integrations/discord/client_test.exs

```diff
@@ -0,0 +1,130 @@
+defmodule Lux.Integrations.Discord.ClientTest do
```

Comment:
Nice! It would be greta to have at least one super basic integration test that actually hits discord.